spec/std/data/test_file.ini

```diff
@@ -1,8 +1,8 @@
 [general]
-log_level = DEBUG
+log_level = D
```

Comment:
It's probably better not to change existing test data. You don't really need to use this file.

Reply:
That's for `Char` type testing, that should be converted to a `String`.